### DISCUSSION TITLE
Infobox locations versus Impacted locations from Wikipedia articles

### DISCUSSION THREAD

**Discussion by koffiworou** [VUB-HYDR/Wikimpacts#56] [Ideas]
The general rule is to map a location with at least one category of impacts, and time information. 
Sometimes, the large-scale location information is provided by the Wikipedia infobox. However, all the locations in that infobox are not necessarily associated with an impact. 
Should we consider the overall location information from the wikipedia infobox for a main event? Or should we consider only the locations that have impact and time information?
Here is an example: 

Event ID 90

https://en.wikipedia.org/wiki/Typhoon_Kujira_(2009)

The area affected, according to the info box are the Philippines and the Russian Far East.
However, the impact information in the text is related to the Philippines.

**Comment by liniiiiii** [comment]
Just have a discussion with Mariana, it's better to keep the Russia in the gold data, for evaluation process, and in the final database, we will check the flequncy of the location, and if it only appear once, which means, the impact may not associated, and we can flag them out with some manual double checking
